echopop/analysis.py

```diff
@@ -0,0 +1,600 @@
+"""
+General analysis orchestration functions that bundle related functions and procedures
+"""
+
+import warnings
+
+import numpy as np
+import pandas as pd
+
+from .acoustics import nasc_to_biomass, summarize_sigma_bs
+from .biology import (
+    distribute_length_age,
+    filter_species,
+    fit_length_weight_relationship,
+    fit_length_weights,
+    number_proportions,
+    partition_transect_age,
+    quantize_number_counts,
+    quantize_weights,
+    weight_proportions,
+)
+from .spatial.krige import kriging
+from .spatial.mesh import crop_mesh, mesh_to_transects, stratify_mesh
+from .spatial.projection import transform_geometry
+from .spatial.transect import (
+    edit_transect_columns,
+    save_transect_coordinates,
+    summarize_transect_strata,
+    transect_distance,
+)
+from .statistics import stratified_transect_statistic
+
+
+def process_transect_data(
+    input_dict: dict, analysis_dict: dict, settings_dict: dict, configuration_dict: dict
+) -> dict:
+    """
+    Calculates stratified mean statistics for a set of transects
```

Comment:
This seems to a result of copy-pasting? Would be good to write out what the outputs are since "process" is a pretty general term.

Reply:
I'm somewhat unclear on what text in this line seems to be the result of copy-pasting? I agree on the latter point though!

Reply:
"Calculates stratified mean statistics for a set of transects" ? clearly not the intent of this function?